src/docs/admin/references/NodeSourceInfrastructureReference.adoc

```diff
@@ -428,7 +430,7 @@ WARNING: Please do not trigger networking autoconfiguration if you operate ProAc
 Otherwise, a new and distinct VPC will be used to operate the nodes created by the NodeSource, preventing their communication with the Resource Manager.
 
 	- *defaultSecurityGroup:* This parameter receives the ID of the security group to spawn instances into.
- If this parameter does not meet the requirement regarding the providing the provided VPC and subnet, a new security group will be generated.
+ If this parameter does not meet the requirement regarding the providing the provided VPC and subnet, a new security group will be generated by default and will be re-used if the same deployment scenario will be repeated.
```

Comment:
Could you be a bit more specific for "the same deployment scenario"? Is it for the same node source deployment or any deployment in the same region (under aws account)?

Reply:
I mean if with the same configuration and still `If this parameter does not meet the requirement regarding the providing the provided VPC and subnet`. Not easy to say it differently.

Reply:
For any node sources in the same region without VPC, the same security group will always be reused ?